index.js

```diff
@@ -15,12 +16,11 @@ const ResultFormatter = require("./src/process-results/result-formatter")
 var AWS = require("aws-sdk");
 
 var publish = function(name, data, extension, options, callback) {
-  if (options.debug) console.log("[" + name + "] Publishing to " + config.aws.bucket + "...");
+  winston.debug("[" + name + "] Publishing to " + config.aws.bucket + "...")
 
   var mime = {".json": "application/json", ".csv": "text/csv"};
-  //console.log(data);
   zlib.gzip(data, function(err, compressed) {
-    if (err) return console.log("ERROR AFTER GZIP: " + err);
+    winston.error("ERROR AFTER GZIP:", err);
```

Comment:
I couldn't find this in a quick search of the `winston` docs, but this will only fire if `err` is truth-y, right?